cmd/minishift/cmd/start.go

```diff
@@ -35,15 +39,14 @@ import (
 	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
 	"github.com/minishift/minishift/pkg/minishift/hostfolder"
 	"github.com/minishift/minishift/pkg/minishift/provisioner"
+	"github.com/minishift/minishift/pkg/minishift/registration"
 	"github.com/minishift/minishift/pkg/util"
+	pkgUtil "github.com/minishift/minishift/pkg/util"
```

Comment:
this is outside of minishift pkg and part of pkg/util instead pkg/minishift/util so I used this. Same what I used in previous PR https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/delete.go#L29 let me know if you still think this should be `minishiftUtil` so I wll change it to other occurrence also.